### ISSUE TITLE
erroneous 403 error on `publish` for scoped packages

### ISSUE THREAD

**Issue by ashleygwilliams** [npm/registry-issue-archive#9]
i am reliably seeing 403 error messages on `npm publish --access=public` for scoped packages. despite the error, the package is successfully published.

npm: 2.15.0
node: 4.4.2
## to reproduce
1. npm init --scope=<whatever>
2. npm publish --access=public

you will see a 403 error in the terminal, however, the package will be successfully published to the registry and appear on the website.

may be related to on-going issues re: https://github.com/npm/registry/issues/2

**Comment by chrisdickinson** [comment]
@ashleygwilliams Does this look like what you're seeing?

```
npm ERR! publish Failed PUT 403
npm ERR! Darwin 15.4.0
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "publish" "--access=public"
npm ERR! node v5.4.1
npm ERR! npm  v3.8.7
npm ERR! code E403

npm ERR! "You cannot publish over the previously published version 0.0.1." : @chrisdickinson/test-000002
npm ERR! 
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/chris/womp/romp/npm-debug.log
```

**Comment by chrisdickinson** [comment]
OK! I have tracked down the issue and fixed it. This is fallout from yesterday's outage — in particular a host had a misconfigured cache URL, and was returning an (erroneous) 500 even though the publish succeeded. This 500 filtered down to `validate-and-store`, the service that wraps couch and handles validating and publishing documents to couch, which forwarded the 500 to the CLI, which immediately retried and got a 403 (because the first publish succeeded.) This is fixed now and we are taking steps to ensure that similar errors don't occur in the future.
